src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService base = new UserService();
+        for (int i = 0; i < base.getSize(); i++) {
+            if (Objects.equals(email, base.getMail(i))) {
+                if (Objects.equals(password, base.getPass(i))) {
+                    return true;
+                }
+            }
+        }
         return false;
```

Comment:
you should use the findByEmail method
```suggestion
        for (int i = 0; i < base.getSize(); i++) {
            if (Objects.equals(email, base.getMail(i))) {
                if (Objects.equals(password, base.getPass(i))) {
                    return true;
                }
            }
        }
        return false;
```